src/elements/element.rectangle.js

```diff
@@ -54,96 +54,92 @@ function getBarBounds(bar) {
 	};
 }
 
+function flip(orig, v1, v2) {
+	return orig === v1 ? v2 : orig === v2 ? v1 : orig;
+}
+
+function parseBorderSkipped(bar) {
+	var vm = bar._view;
+	var vertical = isVertical(bar);
+	var borderSkipped = valueOrDefault(vm.borderSkipped, vertical ? 'bottom' : 'left');
```

Comment:
It looks like it's no longer supporting an option of no skipped border. I think that would be a good option to support because it will be needed for https://github.com/chartjs/Chart.js/pull/6056 at the very least.

Reply:
It does, only `undefined` falls to default, any other value (false, null, '', 'whatever') goes through. And in `parseBorderWidth`, that value is used as key that is set to 0. So `border['whatever']` is set to 0.

Validation of value provided could be added, but I don't think its really needed.

Reply:
Let's add skipping no border to the documentation

Reply:
There is `false` added to list of possible values, isn't that enough?

Reply:
It would be clearer if it is described explicitly.

Reply:
This line disappeared in the latest version because after talking with @kurkle, it seems more consistent to consider all falsy (or invalid) values as disabling the border skipping feature. The rectangle element already provides a default (`borderSkipped: 'bottom'`) and the horizontal bar controller as well (`borderSkipped: 'left'`).

Hitting this line with `borderSkipped: undefined` would mean that the user explicitly set `undefined` in all defaults, in which case it should be fine to consider he want to not skip any border.

Reply:
Though, in the docs, I would promote only `false` as the way to disable the border skip feature.

Reply:
Adding `false` is enough. GitHub cut off part of the code for me. Sorry about the confusion